lib/mustache/parser.rb

```diff
@@ -298,6 +298,15 @@ def scan_tag_block content, fetch, padding, pre_match_position
     end
     alias_method :'scan_tag_#', :scan_tag_block
 
+    def scan_tag_blockvar content, fetch, padding, pre_match_position
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unused method argument - `padding`. If it's necessary, use `_` or `_padding` as an argument name to indicate that it won't be used.](https://www.codacy.com/app/rokusu/mustache/file/1131006120/issues/source?bid=2903613&fileBranchId=2904056#l301)